kubernetes/ansible/roles/logging/templates/fluentd-elasticsearch.yaml

```diff
@@ -130,10 +130,17 @@ extraConfigMaps:
       remove_key_name_field false
         <parse>
           @type regexp
-          expression ^(?<remote>[^ ]*) (?<host>[^ ]*) (?<user>[^ ]*) \[(?<time>[^\]]*)\] "(?<method>\S+)(?: +(?<path>[^\"]*?)(?: +\S*)?)?" (?<status>[^ ]*) (?<size>[^ ]*) (?<request_time>[^ ]+) (?<upstream_response_time>[^ ]+) (?<pipe>\.*) (?:"(?<referer>[^\"]*)" "(?<agent>[^\"]*)" "(?<traceid>[^\"]*)" "(?<deviceid>[^\"]*)" "(?<channelid>[^\"]*)" "(?<appid>[^\"]*)"(?:\s+(?<http_x_forwarded_for>[^ ]+))?)?$ 
+          expression ^(?<cip>[^ ]*) (?<host>[^ ]*) (?<uid>[^ ]*) \[(?<time>[^\]]*)\] "(?<verb>\S+)(?: +(?<path>[^\"]*?)(?: +\S*)?)?" (?<rscode>[^ ]*) (?<rqsize>[^ ]*) (?<rssize>[^ ]*) (?<rqtime>[^ ]+) (?<rstime>[^ ]+) (?<pipe>\.*) (?:"(?<referer>[^\"]*)" "(?<agent>[^\"]*)" "(?<tid>[^\"]*)" "(?<did>[^\"]*)" "(?<cid>[^\"]*)" "(?<appid>[^\"]*)"(?:\s+(?<http_x_forwarded_for>[^ ]+))?)?$ 
```

Comment:
@Kaali09 Please add a comment here on the mapping like this
# field mappings with nginx logs and kibana index as below -
# cip = remote, host = host, uid = user, time = time
and so on..